test/parallel/test-http-get-pipeline-problem.js

```diff
@@ -38,13 +39,15 @@ const image = fixtures.readSync('/person.jpg');
 console.log(`image.length = ${image.length}`);
 
 const total = 10;
-let requests = 0;
-let responses = 0;
+const requestCountdown = new Countdown(total, common.mustCall(() =>
+  server.close()
+));
+const responsecountdown = new Countdown(total, common.mustCall(() =>
```

Comment:
Could you update this `const` to be `camelCase` please?

Reply:
@apapirovski Done. Thanks!!